src/components/SkeletonLoader/SkeletonLoader.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsxImportSource theme-ui */
+import React from "react"
+import ContentLoader from "react-content-loader"
+
+export interface SkeletonLoaderProps {
+  variant?: string
```

Comment:
S: We should type this with all the available variants, instead of just `string`